app.html

```diff
@@ -148,14 +139,19 @@
 scene.add(directionalLight);
 
 // bootstrap
+var angle = 5;
```

Comment:
These could be more descriptive, since they specifically apply to the camera. They could also be more tightly scoped.